The code using url_drag_types[] is #if 0 enclosed, so do
the same for the array to avoid gcc6 error -Wunused-const-variable=

--- src/nemo-notebook.c	2016-03-07 18:19:50.000000000 +0100
+++ src/nemo-notebook.c	2016-09-13 01:00:15.102303362 +0200
@@ -51,11 +51,13 @@
 static void nemo_notebook_remove	 (GtkContainer *container,
 					  GtkWidget *tab_widget);
 
+#if 0
 static const GtkTargetEntry url_drag_types[] =
 {
 	{ NEMO_ICON_DND_GNOME_ICON_LIST_TYPE, 0, NEMO_ICON_DND_GNOME_ICON_LIST },
 	{ NEMO_ICON_DND_URI_LIST_TYPE, 0, NEMO_ICON_DND_URI_LIST },
 };
+#endif
 
 enum
 {
